MapboxNavigation/Style.swift

```diff
@@ -512,3 +514,51 @@ class ManeuverContainerView: UIView {
         }
     }
 }
+
+/// :nodoc:
+@objc(MBStatusView)
+class StatusView: UIView {
+    @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet weak var textLabel: UILabel!
+    @IBOutlet weak var topConstraint: NSLayoutConstraint!
+    
+    func show(_ title: String, showSpinner: Bool, duration: TimeInterval) {
+        textLabel.text = title
+        activityIndicatorView.isHidden = !showSpinner
+        activityIndicatorView.startAnimating()
+        
+        show()
```

Comment:
Fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/468/commits/b6a51bb351e8d7c5f3848acfe27f4122456934cd